packages/client/src/store/forumStore.ts

```diff
@@ -71,4 +80,38 @@ export const useTopicStore = create<ITopicStore>()((set, get) => ({
     const message = await ApiClient.createMessage(body);
     set({ messages: [...get().messages, message.data] });
   },
+  likeMessage: async (body: ILikeMessage) => {
+    try {
+      let message = await ApiClient.likeMessage(body.id, body.userId);
+      if (message.data.dislikes.includes(body.userId)) {
+        message = await ApiClient.dislikeMessage(body.id, body.userId);
+      }
+      const messages = get().messages.map((mes) => {
+        if (mes.id === body.id) {
+          return message.data;
+        }
+        return mes;
+      });
+      set({ messages: messages });
+    } catch (error: unknown) {
+      addToast((error as IEError).response.data.reason);
+    }
+  },
```

Comment:
А вот тут имхо как раз можно воспользоваться деструктуризацией:
```suggestion
  likeMessage: async ({ id, userId }: ILikeMessage) => {
    try {
      let message = await ApiClient.likeMessage(id, userId);
      if (message.data.dislikes.includes(userId)) {
        message = await ApiClient.dislikeMessage(id, userId);
      }
      const messages = get().messages.map((mes) => {
        if (mes.id === id) {
          return message.data;
        }
        return mes;
      });
      set({ messages: messages });
    } catch (error: unknown) {
      addToast((error as IEError).response.data.reason);
    }
  },
```

Reply:
деструктурируй меня полностью